TG.Backend/Data/Validation/AppUserChangePasswordDTOValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+
+namespace TG.Backend.Data.Validation
+{
+    public class AppUserChangePasswordDTOValidator : AbstractValidator<AppUserChangePasswordDTO>
+    {
+        public AppUserChangePasswordDTOValidator()
+        {
+            RuleFor(x => x.Email).EmailAddress().NotEmpty();
```

Comment:
Maybe stupid thing, but consider using .Funtion() in next line for better reading of code :D